tests/external_aiobotocore/test_bedrock_chat_completion.py

```diff
@@ -19,6 +19,7 @@
 import botocore.eventstream
 import botocore.exceptions
 import pytest
+from conftest import BOTOCORE_VERSION  # pylint: disable=E0611
```

Comment:
Can we drop the pylint disable on this import?

Reply:
I'll round up all the pylint and other weird noqa markers in a big pass at some point